### ISSUE TITLE
Create a new issue template: Initiative Review

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/civic-opportunity#162]
### Overview
We need a new template for evaluating initiatives to determine status, and next steps if any so that we can display, prioritize and measure our initiatives.

### Action Items
- [ ] Draft a template in the comments below
- [ ] Get signoff by all team members that they understand how it works
- [ ] Get signoff by Bonnie that it meets the Overview objective

### Resources/Instructions
Admins on the repo can add new templates to the repo, if you don't see the gear at the top of the repo
![settings](https://user-images.githubusercontent.com/37763229/126530971-3dcbcb7a-7e02-40a4-b163-a4d7519bc8f9.png)
and then the button marked ""
![templates button](https://user-images.githubusercontent.com/37763229/126531469-33516a97-5687-4c70-adef-706ff8bdf094.png)

**Comment by ExperimentsInHonesty** [comment]
_There will likely be more Action Items added to this template after we have used it once.  So once the template is approved and created, please don't make all the initiative issues at once.  Make 1 and then we can revaluate after we have used it through the defined steps._

---
name: Initiative Review
about: template for evaluating initiatives to determine status, and next steps if any so that we can display, prioritize and measure our initiatives.
title: Initiative Review: [INSERT NAME OF INITATIVE]
labels: Initiative Review, role: UI/UX research
assignees: ''
---

## Overview
We need to Review the [INSERT NAME OF INITATIVE] to determine determine status, and next steps, if any, so that we can display, prioritize and measure impact of this initiative

### Action Items
- [ ] UI/UX Research Identifies the assets (including documentation) for this initiative and add them to the Resources section below (they might be in wiki, google drive, another issue, etc.).  Please be thorough 
   - [ ] remove the label "role: UI/UX research" and add the label "role: org" and put into the prioritized backlog
- [ ] Org rep self assigns, reviews and adds any additional notes (including Status)
   - [ ] add the label "role: UX" and put into the prioritized backlog for meeting scheduling
- [ ] UI/UX Research and Org rep meet to discuss next steps

### Resources

**Comment by ExperimentsInHonesty** [comment]
@alexilee @Arjayellis Please have the whole team review this to see if they have anything they want to add, clarify, etc.  - notes, including signoff if applicable should be noted in the comments here.

**Comment by Arjayellis** [comment]
@patrickm-tan @Jackie-UX please see above and add any feedback/comments as necessary

**Comment by ExperimentsInHonesty** [comment]
How will we prioritize initiatives?  Please brainstorm below:
- Time
  - Currently in constant, must be kept updated
  - Currently in occasional use, update as needed
  - Future use, define when
  - On hold, why
  - Closed, will not be used again - needs documenting
  - Closed, will not be used again documented
- Impact (directly quantified by how it meets the Objective of Civic Opportunity Project [one sheet](https://docs.google.com/document/d/19Ttc0mkzkB6oENoO45oY3iosrX2zPk9w/edit))
  - 
- Effort
  -

**Comment by alexilee** [comment]
Team brainstormed a few ideas:
- Stakeholder Input (How important does Org rep believe an initiative to be?)
- Effort (t-shirt sizing)
- Impact (rough rating system 1-5?)
- Team Bandwidth - do we have the right people to execute on the initiative?

**Comment by alexilee** [comment]
Created issue template [Review Initiative](https://github.com/hackforla/civic-opportunity/issues/163).

**Comment by Arjayellis** [comment]
How do we assess initiatives in order to assign status and prioritize:

- Status
  - Active (at least one member is actively working on it)
  - On hold (no one is working on it, but we plan to at a future date)
    - Why?
    - Next steps
  - Closed (no one is working on it, and we do not plan to work it moving forward) 
    - Why?

*** Can add whether or not each initiative needs to be updated as necessary

- Impact (1-5 rating?)
  - Reach (how many people is it affecting)
  - Does it affect one or multiple parts of the organization
  - How many types of stakeholders does it benefit

- Effort (1-5 rating?)
  - How much time will it take
  - How many people are required to complete it (tasks, approvals, etc)
  - How many steps are included
  - How technically difficult is it (dependencies, technologies, etc) 

- Stakeholder Input
  - Org Rep - aka, Bonnie
